third_party/terraform/tests/resource_healthcare_fhir_store_test.go.erb

```diff
@@ -104,6 +104,17 @@ func TestAccHealthcareFhirStore_basic(t *testing.T) {
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
+			{
+				Config: testGoogleHealthcareFhirStore_basic(fhirStoreName, datasetName),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
This check is actually covered by the combination of some default testing stuff (making sure there's an empty plan after applying) and the import test below it (making sure states match when a resource is created/updated and when it's imported), so you can remove it.

Reply:
Done.